tests/test_models_unet.py

```diff
@@ -43,15 +43,16 @@ def dummy_input(self):
 
     @property
     def input_shape(self):
-        return (3, 32, 32)
+        return (3, 31, 31)
```

Comment:
Would it maybe be better to add a new test (would be nice to keep testing the default size of 32)